SNIPS/snip-18.md

```diff
@@ -0,0 +1,163 @@
+---
+snip: 18
+title: Staking’s First Stage on Starknet
+author: Natan Granit <natan@starkware.co>
+discussions-to:
```

Comment:
add link to forum thread https://community.starknet.io/t/snip-18-staking-s-first-stage-on-starknet/